fiftyone/core/annotation/validate_label_schemas.py

```
@@ -427,77 +457,100 @@ def _raise_unknown_setting_error(name, field_name):
+                field,
+                field_name,
                 path,
                 subfields,
+                allow_new_attrs=allow_new_attrs,
             )
+
+            if attr in validated:
+                raise ValueError(
+                    f"'{foac.ATTRIBUTES}' setting for field '{field_name}' "
+                    f"ha duplicate '{attr}' settings"
+                )
```

Ruff (0.14.14)

Abstract `raise` to an inner function

(TRY301)